src/Route.elm

```diff
@@ -20,6 +22,12 @@ toString route =
         Index ->
             "/"
 
+        CaseStudy ->
+            "/case-study"
```

Comment:
I think we probably want to call this route something other than case-study. I'm speaking to @undividual about it here #36

Reply:
No, just put it in like this - we'll make the decision in that other ticket.